maintenance_mode/views.py

```diff
@@ -13,7 +13,7 @@ def maintenance_mode_off(request):
     if request.user.is_superuser:
         set_maintenance_mode(False)
 
-    return HttpResponseRedirect('/')
+    return HttpResponseRedirect(request.META['SCRIPT_NAME'] or '/')
```

Comment:
`request.META` is a dictionary, a better and `KeyError` proof approach would be to use the `dict.get` method with a default value: 
```python
return HttpResponseRedirect(request.META.get('SCRIPT_NAME', '/'))
```